### ISSUE TITLE
Unexpected error message

### ISSUE THREAD

**Issue by MikeVitik** [palantir/tslint#417]
Hi. I've got the error:
TsLint: expected an assignment or function call

for the first line of the following code:

```ts
    export function makeValuesArray(values: (string | { value: string; displayValue: string; })[]) {
        return values.map((item) => {
            if(typeof item === "string") {
                var strItem = <string>item;
                return { value: strItem, displayValue: strItem.charAt(0).toUpperCase() + strItem.slice(1) };
            } else {
                return item;
            }
        });
    }
```

**Comment by leeavital** [comment]
What version and rules are you using? I can't reproduce this.

**Comment by adidahiya** [comment]
closing due to inactivity

**Comment by MikeVitik** [comment]
Sorry for delay. Error on my side. I use VS with webessential plugins and
it use old version.

чт, 18 июня 2015, 20:56, Adi Dahiya <notifications@github.com>:
